libraries/chain/include/eosio/chain/hotstuff/hotstuff.hpp

```diff
@@ -93,14 +93,18 @@ namespace eosio::chain {
 
       // thread safe
       bool is_quorum_met() const;
+      static bool is_quorum_met(state_t s) {
+         return s == state_t::weak_achieved || s == state_t::weak_final || s == state_t::strong;
```

Comment:
In normal operations, `s == state_t::strong` should be more common. Can it be placed at the beginning?